packages/botonic-nlp/tests/dataset/dataset.test.ts

```diff
@@ -0,0 +1,30 @@
+import { Dataset } from '../../src/dataset/dataset'
+
+describe('Dataset', () => {
+  const sut = new Dataset(
+    ['BuyProduct', 'ReturnProduct'],
+    ['product', 'material'],
+    [
+      { text: 'I want to buy this', class: 'BuyProduct', entities: [] },
+      { text: 'I want to return this', class: 'ReturnProduct', entities: [] },
+      {
+        text: 'I would like to return it',
+        class: 'ReturnProduct',
+        entities: [],
+      },
+      { text: 'I would like to buy it', class: 'BuyProduct', entities: [] },
+    ]
+  )
```

Comment:
Just a comment of how I like to make/see tests (do we have any "best practices" documentation about how to structure tests @dpinol?)

I think it's a good practice to have all the tests at the beginning of the test file and at the bottom all the datasets or initialization functions. This way you can see right away what are being tested and if all the details will be at the bottom.

In this case, you could create a `getDataset()` function to return this `Dataset` and the describe block would be much cleaner. What do you think @elozano98?

Reply:
Also, why not using the `dataset` of `tools-helper`? You're already loading a dataset there. Is it too big?

Reply:
@asastre see https://www.notion.so/Unit-tests-8515f1d36352406ea461d06c08b2ee36#5b3d48716fac4ef6b76912f4e135a9e4

> If you need to repeat a given complex function call or data structure many times, create a helper function or constant as close and customized as possible to the test. 

I think the test follows the best practice which makes it readable because the data is close to the test

Reply:
Thanks for the link and the explanation @dpinol !